be/src/runtime/data_stream_recvr.h

```diff
@@ -158,6 +157,9 @@ class DataStreamRecvr {
     // total buffer limit.
     bool exceeds_limit(int chunk_size) { return _num_buffered_bytes + chunk_size > _total_buffer_limit; }
 
+    // Return a metrics for current rpc in round-robin manner.
+    Metrics& get_metrics_round_robin() { return _metrics[_rpc_round_roubin_index++ % _metrics.size()]; }
```

Comment:
should it overflow so get negative value, resulting crash.

Reply:
`-1 % 5` is `-1`, I thought it woud be `1`. I'll update.

Reply:
I've changed the type of _rpc_round_roubin_index from `int32_t` to `size_t`